layersvt/screenshot.cpp

```diff
@@ -678,9 +678,16 @@ static void writePPM(const char *filename, VkImage image1) {
 
     // Set up the command buffer.  We get a command buffer from a pool we saved
     // in a hooked function, which would be the application's pool.
+    if (deviceMap[device]->commandPools.empty()) {
+        assert(!deviceMap[device]->commandPools.empty());
```

Comment:
Do we want this assert here? If there isn't a command pool to use, this will crash the process when compiled Debug, but will silently do nothing in Release mode.  Maybe the behavior should be the same for Release/Debug?